### ISSUE TITLE
Ignore *.sqlite files to avoid restarting when using cargo watch

### ISSUE THREAD

**Issue by ycharron** [loco-rs/loco#774]
**Description**

As per the documentation FAQ (How can I automatically reload code?). Auto-reload is run as `$ cargo-watch -x check  -s 'cargo loco start'`.

When I run the above command, the application reloads continuously ~every 1/2 second.

**To Reproduce**

I'm using a fresh (non-modified) SaaS, Server generated cargo app.

**Expected Behavior**

Only refreshes when code changes.

**Environment:**

Debian 12

**Comment by jondot** [comment]
Thanks for this report
Can you run cargo-watch with `--debug`?

**Comment by ycharron** [comment]
I'm new to Rust thus still learning about its ecosystem, but here's what I found:

- As previously guesstimated, the issue is related to the loco app generating this (sqlite) statement every second: e.g. 2024-09-20T17:04:53.378897Z INFO sqlx::query: summary="PRAGMA foreign_keys = ON; …" db.statement="\n\nPRAGMA foreign_keys = ON;\n" rows_affected=0 rows_returned=0 elapsed=34.97µs elapsed_secs=3.497e-5
- Said statement updates the .sqlite file which in turn triggers the cargo watch to reload.
- While reading the cargo watch documentation, files in .gitignore should be ignored by default. *.sqlite is in .gitignore, but unsure why it's not working for *.sqlite.
- Adding --ignore '*.sqlite' to cargo watch resolves the problem.

I've used SQLite before, but I don't remember the framework I used at the time executing the above, (or any) statement every second. Not sure if that's expected or not.
